opm/autodiff/SimulatorFullyImplicitBlackoilOutput.cpp

```diff
@@ -52,6 +57,17 @@ namespace Opm
 {
 
 
+
```

Comment:
I'm usually a big fan of more whitespace between functions, but this seemed a bit excessive even to me :-)